engine/runtime/src/test/java/org/enso/interpreter/test/ValuesGenerator.java

```diff
@@ -369,7 +369,7 @@ public List<Value> numbers() {
     if (languages.contains(Language.ENSO)) {
       collect.add(v(null, "", "42").type());
       collect.add(v(null, "", "6.7").type());
-      collect.add(v(null, "", "40321 * 43202").type());
+      collect.add(v(null, "import Standard.Base.Data.Numbers", "40321 * 43202").type());
```

Comment:
I wonder if we should add a TODO mentioning #8852 here, so that we remember to later remove this import?

Reply:
This places is no special. Adding TODO to all places would mean adding a lot of TODOs...